src/components/dialogs/network-modifications/tabular-modification/tabular-modification-dialog.js

```diff
@@ -105,10 +118,18 @@ const TabularModificationDialog = ({
                 };
                 Object.keys(row).forEach((key) => {
                     const value = row[key];
-                    if (key === 'equipmentId') {
-                        modification[key] = value;
-                    } else {
-                        modification[key] = toModificationOperation(value);
+                    switch (key) {
+                        case 'equipmentId':
+                            modification[key] = value;
+                            break;
+                        case 'energySource':
+                            modification[key] = toModificationOperation(
```

Comment:
you can't use value directly here ?
modification[key] = toModificationOperation(value) ?